orchagent/pfcwdorch.cpp

```diff
@@ -219,6 +225,10 @@ task_process_status PfcWdOrch<DropHandler, ForwardHandler>::createEntry(const st
                     SWSS_LOG_ERROR("Invalid PFC Watchdog action %s", value.c_str());
                     return task_process_status::task_invalid_entry;
                 }
+                if ((m_platform == CISCO_8000_PLATFORM_SUBSTRING) && (action == PfcWdAction::PFC_WD_ACTION_FORWARD)) {
```

Comment:
@neethajohn is this ok not to have forward action ?

Reply:
@abdosi, it is okay to not have forward action